core/src/avm2/error.rs

```diff
@@ -571,6 +571,26 @@ pub fn make_error_2008<'gc>(activation: &mut Activation<'_, 'gc>, param_name: &s
     }
 }
 
+#[inline(never)]
+#[cold]
+pub fn make_error_2012<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    class_name: impl ToString,
```

Comment:
Very small nit: it would make more sense to take `impl Display` instead

Reply:
That makes sense, thanks!